src/Knet.jl

```diff
@@ -26,6 +26,7 @@ include("broadcast.jl");        # elementwise broadcasting operations
 include("reduction.jl");        # sum, max, mean, etc.
 include("linalg.jl");           export mat # matmul, axpy!, transpose, (i)permutedims
 include("conv.jl");             export conv4, pool, deconv4, unpool
+include("batchnorm.jl");        export batchnorm, BNMoments, bnparam, bnscale, bnbias
```

Comment:
Use a regular lowercase bnmoments() function instead, in case we change our minds about using a struct here in the future?

Reply:
Also, why are we exporting bnscale and bnbias?

Reply:
`bnmoments` seems a better name, since it matches with the `bnparams` conventionally. So  it is reasonable to make this change. Should user still know moments is a struct, or should we add accessor methods like `mean(moments)`, `var(moments)` etc.? 

I exported bnscale and bnbias for easy access to g and b, but I think I'll remove these exports, they just return first and second halves of the same array, without even reshaping.